src/test/kotlin/graphics/scenery/tests/examples/basic/IntersectionExample.kt

```diff
@@ -25,10 +27,13 @@ class IntersectionExample: SceneryBase("IntersectionExample") {
             metallic = 1.0f
         }
 
-        val box = Box(Vector3f(1.0f, 1.0f, 1.0f))
+//        val box = Box(Random.random3DVectorFromRange(0.1f, 1.0f))
+        val box = Icosphere(1.0f, 2)
+//        val box = Box()
         box.name = "le box du win"
 
-        val box2 = Box(Vector3f(1.0f, 1.0f, 1.0f))
+//        val box2 = Box()
+        val box2 = Box(Random.random3DVectorFromRange(0.1f, 1.0f))
```

Comment:
With these dimensions, box2 is sometimes too small to cause any intersections. Maybe a range minimum of 0.2 would be better here.